anchore_engine/services/policy_engine/__init__.py

```diff
@@ -350,6 +352,33 @@ def handle_feed_sync_trigger(*args, **kwargs):
     return True
 
 
+def handle_grypedb_sync(*args, **kwargs):
+    """
+    Calls function to run GrypeDBSyncTask
+
+    :param args:
+    :param kwargs:
+    :return:
+    """
+    system_user = _system_creds()
+    if "GrypeDBSyncTask" not in locals():
+        from anchore_engine.services.policy_engine.engine.tasks import GrypeDBSyncTask
+
+    logger.info("init args: {}".format(kwargs))
+    cycle_time = kwargs["mythread"]["cycle_timer"]
+
+    while True:
```

Comment:
I probably lack a ton of understanding on how this is supposed to work, but is this really meant to run for ever?

Reply:
Yeah this is one of the handlers for the monitors that is supposed to run every interval defined in the monitor's cycle_timer. @nightfurys has a better understanding of how these monitors work and she said that if the function were to return, the monitor would call it again. I followed the pattern implemented elsewhere in the file which runs them in this infinite loop